SCFO/Assignment_4/TestArrOperation.java

```diff
@@ -0,0 +1,123 @@
+import static org.junit.Assert.*;
+import org.junit.Test;
+public class TestArrOperation
+{
+
+
+	//This method is used to test the code of count the clumps in an array.
+
+	@Test
+	public void testForClump()
+	{
+		try
+		{
+			ArrOperation obj1 = new ArrOperation();
+			int resultClump = obj1.countOfClumps(new int[]{1, 2 ,2, 3, 4, 4});
+			assertEquals( 2 , resultClump);
+
+			resultClump = obj1.countOfClumps(new int[]{1,1,2,1,1});
+			assertEquals( 2 , resultClump);
+
+			resultClump = obj1.countOfClumps(new int[]{1,1,1,1,1});
+			assertEquals(1 , resultClump);
+
+			resultClump = obj1.countOfClumps(new int[]{0,0,0,0,0});
+			assertEquals(1 , resultClump);
+		}
+
+		catch(AssertionError ex) 
+		{
+			System.out.println("Your array is empty in test for clumps.");
+		}
+
+
+	}
+
+	//This method is used to test the code of Split Array Index which means position at which array can be split.
```

Comment:
Please use documentation comments for function descriptions, as I have mentioned before as well.